src/components/Rating/selectEmoji.js

```diff
@@ -0,0 +1,33 @@
+const SelectEmoji = (ratingEmoji) => {
```

Comment:
I think we should move this to another place such as helper. IMO, components should return React component instead of string.

Also, this can just be a simple object mapper
```js
const emojiMapper = {
    heart: '❤️',
    smiley: '🙂',
    star: '⭐',
    money: '💰',
    thumb: '👍🏻'
};

export default emojiMapper;
```
Then you can use just like this
```js
import emojiMapper from ...

const emoji = emojiMapper[ratingEmoji];
```

What do you think?

Reply:
@rafayet-monon I agreed with @gutakk I think it would be simpler to have this as a helper and use the object instead of a function with switch case 😄